doc/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+
+## What issue is this change attempting to solve?
+
+## How does this change solve the problem? Why is this the best approach?
+
+## How can we verify this works as expected?
+
+## What feedback would you like, if any?
```

Comment:
I don't think this is necessary. It's not a must (just as it says).

Reply:
Yes, I was thinking it might useful to set expectations.

TBH I didn't use templates before, my guess is that if we set this up when you open a PR you will see this list of items already filled in.
So they are not making anything "required", they are more like a guide. If you want to leave them empty(or remove them) I think it's still possible.

Reply:
then we can remove "if any"
but honestly, I don't think this item is necessary

Reply:
Removed!